pkg/container/resolver_test.go

```diff
@@ -104,8 +111,88 @@ func TestResolverFail(t *testing.T) {
 		common.ToPtr(""),
 		common.ToPtr(false),
 		false,
+		nil,
 	})
 	specs, err = resolver.Finish()
 	assert.Error(t, err)
 	assert.Len(t, specs, 0)
 }
+
+func TestResolverLocalManifest(t *testing.T) {
+	currentUser, err := user.Current()
+	if err != nil {
+		t.Fatalf("Failed to get current user: %v", err)
+	}
+
+	if currentUser.Uid != "0" {
+		t.Skip("User is not root, skipping test")
+	}
+
+	_, err = exec.LookPath("podman")
+	if err != nil {
+		t.Skip("Podman not available, skipping test")
+	}
+
+	containerFile, err := os.CreateTemp("", "Containerfile")
+	if err != nil {
+		t.Fatalf("Failed to create tempfile: %v", err)
+	}
+	defer os.Remove(containerFile.Name())
+
+	tmpStorage, err := os.MkdirTemp("", "storage")
```

Comment:
Same as above, `t.TempDir()` could be used here

Reply:
I added a comment... because of overlay storage `t.TempDir` doesn't get cleaned up properly and so the test fails.
So I left it as is

Reply:
Interesting! I (strongly) suspect our own cleanup below (defer os.Remove()) also fails but because we ignore the error we just never noticed.

Reply:
I just tested and you are correct

Reply:
Okay pushing a fix for this.